google/cloud/alloydb/connector/instance.py

```diff
@@ -55,21 +79,16 @@ def __init__(
         client: AlloyDBClient,
         keys: asyncio.Future[Tuple[rsa.RSAPrivateKey, str]],
     ) -> None:
+
+        self._instance_uri = instance_uri
```

Comment:
let's move setting `self._instance_uri` under the parse function call. This way we parse/validate prior to setting it.

Reply:
Hi @jackwotherspoon, I have reordered the assignment of the class member after validating the uri.